js/tsconfig.json

```diff
@@ -6,6 +6,7 @@
   },
   "compilerOptions": {
     "target": "ESNEXT",
-    "module": "commonjs"
+    "module": "commonjs",
+    "noEmit": true
```

Comment:
Now we can run yarn tsc to check types without accidentally writing a bunch of files.